packages/api/src/graphql/mutation.private.ts

```diff
@@ -774,11 +774,14 @@ export const GraphQLAdminMutation = new GraphQLObjectType<undefined, Context>({
 
         if (!article) throw new NotFound('article', id)
 
-        const articleRevision = Object.assign(article.published ?? article.draft, {
-          slug: '',
-          publishedAt: undefined,
-          updatedAt: undefined
-        })
+        const articleRevision = Object.assign(
+          article.published ?? article.pending ?? article.draft,
```

Comment:
I actually would suggest you change the order. First check if there is a `draft` if not check for `pending` and only then take the `published`. I think this is more intuitively for the users.

Reply:
@roosnic1 good point